MinPlatformPkg/PlatformInit/PlatformInitSmm/PlatformInitMm.c

```diff
@@ -0,0 +1,118 @@
+/** @file
+  This driver will register two callbacks to call fsp's notifies.
```

Comment:
This comment does not appear to be accurate.

Reply:
Something about it being a common driver for the Traditional MM and Standalone MM drivers would be appropriate.

Reply:
Fixed to reflect what the file actually does and also mentions that it's a common file between Traditional and Standalone MM.